backend/src/services/log.ts

```diff
@@ -147,7 +157,8 @@ if (process.env.NODE_ENV !== 'production') {
   // In production environments output plain JSON logs
   streams.push({
     level: config.log.level,
-    stream: process.stdout,
+    type: 'raw',
+    stream: ProductionStream(),
```

Comment:
I'm inclined to say that we should add elasticsearch to our dev environments too!

Reply:
Could you expand on this, are there changes you're suggesting for this PR?